Gemfile.lock

```diff
@@ -217,7 +224,7 @@ GEM
     libv8 (3.16.14.17)
     loofah (2.0.3)
       nokogiri (>= 1.5.9)
-    mail (2.6.6)
+    mail (2.6.4)
```

Comment:
are these version changes to a lower version really necessary? i don't think so

Reply:
If you would like to remove these changes, that would clean it up a bit (and avoid a conflict).
Or do you need a newer prawn? If you do, please rebase on current `master`.